src/app/page.tsx

```diff
@@ -0,0 +1,11 @@
+import Text from "../components/Text";
+import './page.css'
+import { wrapper } from "./page.css";
```

Comment:
import가 중복으로 되어있어서 L2는 제거되어야 할 것 같아요!
```suggestion
import { wrapper } from "./page.css";
```

Reply:
page.css 파일에 스타일이 추가될 때 마다 계속 import를 해줘야 해서 아래와 같이 사용해도 좋은 것 같아요!

```suggestion
import * as styles from "./page.css";
```